runtime/runtime/src/actions.rs

```diff
@@ -308,7 +307,7 @@ pub(crate) fn action_stake(
             stake.public_key.clone(),
             stake.stake,
             #[cfg(feature = "protocol_feature_chunk_only_producers")]
-            is_chunk_only,
+            false,
```

Comment:
do we still need this parameter ? we probably can also remove ValidatorStakeV2

Reply:
Yeah, thought the same: https://nearinc.atlassian.net/browse/CPR-83?focusedCommentId=15564

I guess you suggestion means that we don't need the concept of chunk-only stake, so I'll remove that in a follow-up.

Reply:
https://github.com/near/nearcore/pull/7085